org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2.yml

```diff
@@ -0,0 +1,31 @@
+id: org.freedesktop.LinuxAudio.Plugins.NeuralAmpModelerLV2
+branch: '22.08'
+build-extension: true
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-options:
+  prefix: /app/extensions/Plugins/NeuralAmpModelerLV2
+modules:
+  - name: NeuralAmpModelerLV2
+    buildsystem: cmake
```

Comment:
```suggestion
    buildsystem: cmake-ninja
```
if it works

Reply:
Is it still useful to use `cmake-ninja` here if the build system doesn't use ninja explicitly?

Reply:
cmake will be called with `-G Ninja` and the result is much faster build. However in some rare cases it doesn't work that's why it's an option.